test-apps/presentation-test-app/src/frontend/components/diagnostics-selector/DiagnosticsSelector.tsx

```diff
@@ -31,6 +31,10 @@ export function DiagnosticsSelector(props: DiagnosticsSelectorProps) {
     },
   }), [shouldMeasurePerformance, editorSeverity, devSeverity]);
 
+  React.useEffect(() => {

```

Comment:
`result` is missing as dependency for `useEffect`?

Reply:
Fixed, see #1290